

2023-02-16 15:40:48,710:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:48,712:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:48] "GET / HTTP/1.1" 200 -
2023-02-16 15:40:51,593:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:51,594:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:51] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:53,933:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:53,935:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:53] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:54,749:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:54,750:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:54] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:55,605:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:55,607:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:55] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:56,206:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:56,207:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:56] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:56,725:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:56,726:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:56] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:57,237:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:57,239:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:57] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:57,773:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:57,774:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:57] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:58,299:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:58,300:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:58] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:40:59,087:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:40:59,088:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:40:59] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:42:59,293:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 15:42:59,527:DEBUG:NeuronScrapper:Done
2023-02-16 15:42:59,527:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 15:43:01,334:DEBUG:NeuronScrapper:Done
2023-02-16 15:43:01,334:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 15:43:01,344:DEBUG:NeuronScrapper:Done
2023-02-16 15:43:01,345:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 15:43:01,345:DEBUG:NeuronScrapper:Done
2023-02-16 15:43:01,345:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 15:43:01,368:DEBUG:NeuronScrapper:Done
2023-02-16 15:43:01,369:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 15:43:01,369:DEBUG:NeuronScrapper:Done
2023-02-16 15:43:01,369:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 15:43:01,372:DEBUG:NeuronScrapper:Done
2023-02-16 15:43:01,372:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 15:43:01,373:DEBUG:NeuronScrapper:Done
2023-02-16 15:43:01,374:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 15:43:01,409:DEBUG:NeuronScrapper:Done
2023-02-16 15:43:01,416:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 15:43:01,416:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 15:43:06,885:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:43:06,890:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:43:06] "GET / HTTP/1.1" 200 -
2023-02-16 15:43:09,554:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:43:09,556:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:43:09] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:43:09,646:INFO:root:MongoDB Connection...Done
2023-02-16 15:43:09,646:INFO:root:MongoDB Database...Done
2023-02-16 15:43:09,646:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 15:43:09,646:INFO:root:MongoDB Collection courses ...Created
2023-02-16 15:43:10,964:INFO:root:MongoDB Data Insert ...Done
2023-02-16 15:43:12,979:INFO:root:MongoDB Data Insert ...Done
2023-02-16 15:43:13,768:INFO:root:Connected to MySQL Database...
2023-02-16 15:43:14,128:INFO:root:Created Database...iNeuronDb
2023-02-16 15:43:14,300:INFO:root:Using Database...iNeuronDb
2023-02-16 15:43:14,565:INFO:root:Created Table...courses
2023-02-16 15:43:14,565:INFO:root:Total data to Insert: 450
2023-02-16 15:44:35,271:INFO:root:Total data Inserted in Table: 1
2023-02-16 15:44:35,271:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 15:44:35,271:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:44:35] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 15:44:35,281:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:44:35,286:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:44:35] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:46:31,075:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:46:31,079:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:46:31] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:46:34,954:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:46:34,963:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:46:34] "POST /course_list HTTP/1.1" 200 -
2023-02-16 15:46:37,392:ERROR:root:'latin-1' codec can't encode character '\u2019' in position 1581: ordinal not in range(256)
2023-02-16 15:46:37,392:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:46:37] "POST /course_detail HTTP/1.1" 200 -
2023-02-16 15:50:20,295:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 15:50:20,545:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:20,545:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 15:50:22,850:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:22,850:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 15:50:22,861:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:22,861:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 15:50:22,862:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:22,862:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 15:50:22,886:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:22,886:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 15:50:22,886:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:22,886:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 15:50:22,889:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:22,889:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 15:50:22,892:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:22,892:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 15:50:22,927:DEBUG:NeuronScrapper:Done
2023-02-16 15:50:22,934:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 15:50:22,934:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 15:50:33,037:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:50:33,043:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:50:33] "GET / HTTP/1.1" 200 -
2023-02-16 15:50:35,689:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:50:35,692:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:50:35] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:50:35,774:INFO:root:MongoDB Connection...Done
2023-02-16 15:50:35,774:INFO:root:MongoDB Database...Done
2023-02-16 15:50:35,775:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 15:50:35,775:INFO:root:MongoDB Collection courses ...Created
2023-02-16 15:50:37,011:INFO:root:MongoDB Data Insert ...Done
2023-02-16 15:50:40,241:INFO:root:MongoDB Data Insert ...Done
2023-02-16 15:50:41,163:INFO:root:Connected to MySQL Database...
2023-02-16 15:50:41,523:INFO:root:Created Database...iNeuronDb
2023-02-16 15:50:41,702:INFO:root:Using Database...iNeuronDb
2023-02-16 15:50:41,873:INFO:root:Created Table...courses
2023-02-16 15:50:41,873:INFO:root:Total data to Insert: 450
2023-02-16 15:52:00,743:INFO:root:Total data Inserted in Table: 1
2023-02-16 15:52:00,743:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 15:52:00,743:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:52:00] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 15:52:00,766:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:52:00,773:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:52:00] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:52:47,257:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:52:47,261:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:52:47] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:52:48,865:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:52:48,875:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:52:48] "POST /course_list HTTP/1.1" 200 -
2023-02-16 15:52:51,311:INFO:root:File Name for PDF: 6389b7ff41896d32c199d378.pdf
2023-02-16 15:52:51,313:DEBUG:botocore.hooks:Changing event name from creating-client-class.iot-data to creating-client-class.iot-data-plane
2023-02-16 15:52:51,313:DEBUG:botocore.hooks:Changing event name from before-call.apigateway to before-call.api-gateway
2023-02-16 15:52:51,314:DEBUG:botocore.hooks:Changing event name from request-created.machinelearning.Predict to request-created.machine-learning.Predict
2023-02-16 15:52:51,315:DEBUG:botocore.hooks:Changing event name from before-parameter-build.autoscaling.CreateLaunchConfiguration to before-parameter-build.auto-scaling.CreateLaunchConfiguration
2023-02-16 15:52:51,315:DEBUG:botocore.hooks:Changing event name from before-parameter-build.route53 to before-parameter-build.route-53
2023-02-16 15:52:51,316:DEBUG:botocore.hooks:Changing event name from request-created.cloudsearchdomain.Search to request-created.cloudsearch-domain.Search
2023-02-16 15:52:51,316:DEBUG:botocore.hooks:Changing event name from docs.*.autoscaling.CreateLaunchConfiguration.complete-section to docs.*.auto-scaling.CreateLaunchConfiguration.complete-section
2023-02-16 15:52:51,318:DEBUG:botocore.hooks:Changing event name from before-parameter-build.logs.CreateExportTask to before-parameter-build.cloudwatch-logs.CreateExportTask
2023-02-16 15:52:51,318:DEBUG:botocore.hooks:Changing event name from docs.*.logs.CreateExportTask.complete-section to docs.*.cloudwatch-logs.CreateExportTask.complete-section
2023-02-16 15:52:51,318:DEBUG:botocore.hooks:Changing event name from before-parameter-build.cloudsearchdomain.Search to before-parameter-build.cloudsearch-domain.Search
2023-02-16 15:52:51,318:DEBUG:botocore.hooks:Changing event name from docs.*.cloudsearchdomain.Search.complete-section to docs.*.cloudsearch-domain.Search.complete-section
2023-02-16 15:52:51,320:DEBUG:botocore.utils:IMDS ENDPOINT: http://169.254.169.254/
2023-02-16 15:52:51,321:DEBUG:botocore.credentials:Looking for credentials via: env
2023-02-16 15:52:51,321:DEBUG:botocore.credentials:Looking for credentials via: assume-role
2023-02-16 15:52:51,321:DEBUG:botocore.credentials:Looking for credentials via: assume-role-with-web-identity
2023-02-16 15:52:51,321:DEBUG:botocore.credentials:Looking for credentials via: sso
2023-02-16 15:52:51,321:DEBUG:botocore.credentials:Looking for credentials via: shared-credentials-file
2023-02-16 15:52:51,321:INFO:botocore.credentials:Found credentials in shared credentials file: ~/.aws/credentials
2023-02-16 15:52:51,322:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\endpoints.json
2023-02-16 15:52:51,330:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\sdk-default-configuration.json
2023-02-16 15:52:51,330:DEBUG:botocore.hooks:Event choose-service-name: calling handler <function handle_service_name_alias at 0x000002A91CEEFA60>
2023-02-16 15:52:51,363:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\s3\2006-03-01\service-2.json
2023-02-16 15:52:51,410:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\s3\2006-03-01\endpoint-rule-set-1.json.gz
2023-02-16 15:52:51,414:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\partitions.json
2023-02-16 15:52:51,415:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function add_generate_presigned_post at 0x000002A91CE2F880>
2023-02-16 15:52:51,416:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function lazy_call.<locals>._handler at 0x000002A91E318EA0>
2023-02-16 15:52:51,435:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function add_generate_presigned_url at 0x000002A91CE2F600>
2023-02-16 15:52:51,436:DEBUG:botocore.endpoint:Setting s3 timeout as (60, 60)
2023-02-16 15:52:51,437:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\_retry.json
2023-02-16 15:52:51,438:DEBUG:botocore.client:Registering retry handlers for service: s3
2023-02-16 15:52:51,438:DEBUG:botocore.utils:Registering S3 region redirector handler
2023-02-16 15:52:51,438:DEBUG:s3transfer.utils:Acquiring 0
2023-02-16 15:52:51,439:DEBUG:s3transfer.tasks:UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000002A91D83B550>}) about to wait for the following futures []
2023-02-16 15:52:51,439:DEBUG:s3transfer.tasks:UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000002A91D83B550>}) done waiting for dependent futures
2023-02-16 15:52:51,439:DEBUG:s3transfer.tasks:Executing task UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000002A91D83B550>}) with kwargs {'client': <botocore.client.S3 object at 0x000002A91E09C310>, 'config': <boto3.s3.transfer.TransferConfig object at 0x000002A91D416DD0>, 'osutil': <s3transfer.utils.OSUtils object at 0x000002A91E0AFC10>, 'request_executor': <s3transfer.futures.BoundedExecutor object at 0x000002A91E0AED50>, 'transfer_future': <s3transfer.futures.TransferFuture object at 0x000002A91D83B550>}
2023-02-16 15:52:51,439:DEBUG:s3transfer.futures:Submitting task PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '6389b7ff41896d32c199d378.pdf', 'extra_args': {}}) to executor <s3transfer.futures.BoundedExecutor object at 0x000002A91E0AED50> for transfer request: 0.
2023-02-16 15:52:51,439:DEBUG:s3transfer.utils:Acquiring 0
2023-02-16 15:52:51,440:DEBUG:s3transfer.tasks:PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '6389b7ff41896d32c199d378.pdf', 'extra_args': {}}) about to wait for the following futures []
2023-02-16 15:52:51,440:DEBUG:s3transfer.utils:Releasing acquire 0/None
2023-02-16 15:52:51,440:DEBUG:s3transfer.tasks:PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '6389b7ff41896d32c199d378.pdf', 'extra_args': {}}) done waiting for dependent futures
2023-02-16 15:52:51,440:DEBUG:s3transfer.tasks:Executing task PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '6389b7ff41896d32c199d378.pdf', 'extra_args': {}}) with kwargs {'client': <botocore.client.S3 object at 0x000002A91E09C310>, 'fileobj': <s3transfer.utils.ReadFileChunk object at 0x000002A91D433150>, 'bucket': 'arunpdf', 'key': '6389b7ff41896d32c199d378.pdf', 'extra_args': {}}
2023-02-16 15:52:51,440:DEBUG:botocore.hooks:Event before-endpoint-resolution.s3: calling handler <function customize_endpoint_resolver_builtins at 0x000002A91CF037E0>
2023-02-16 15:52:51,440:DEBUG:botocore.hooks:Event before-endpoint-resolution.s3: calling handler <bound method S3RegionRedirectorv2.redirect_from_cache of <botocore.utils.S3RegionRedirectorv2 object at 0x000002A91E0B8050>>
2023-02-16 15:52:51,441:DEBUG:botocore.regions:Calling endpoint provider with parameters: {'Bucket': 'arunpdf', 'Region': 'ap-northeast-1', 'UseFIPS': False, 'UseDualStack': False, 'ForcePathStyle': False, 'Accelerate': False, 'UseGlobalEndpoint': False, 'DisableMultiRegionAccessPoints': False, 'UseArnRegion': True}
2023-02-16 15:52:51,441:DEBUG:botocore.regions:Endpoint provider result: https://arunpdf.s3.ap-northeast-1.amazonaws.com
2023-02-16 15:52:51,441:DEBUG:botocore.regions:Selecting from endpoint provider's list of auth schemes: "sigv4". User selected auth scheme is: "None"
2023-02-16 15:52:51,441:DEBUG:botocore.regions:Selected auth type "v4" as "v4" with signing context params: {'region': 'ap-northeast-1', 'signing_name': 's3', 'disableDoubleEncoding': True}
2023-02-16 15:52:51,441:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function validate_ascii_metadata at 0x000002A91CF022A0>
2023-02-16 15:52:51,441:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function sse_md5 at 0x000002A91CF01580>
2023-02-16 15:52:51,441:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function convert_body_to_file_like_object at 0x000002A91CF02CA0>
2023-02-16 15:52:51,441:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function validate_bucket_name at 0x000002A91CF014E0>
2023-02-16 15:52:51,441:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function remove_bucket_from_url_paths_from_model at 0x000002A91CF03600>
2023-02-16 15:52:51,441:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <bound method S3RegionRedirectorv2.annotate_request_context of <botocore.utils.S3RegionRedirectorv2 object at 0x000002A91E0B8050>>
2023-02-16 15:52:51,442:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function generate_idempotent_uuid at 0x000002A91CF01300>
2023-02-16 15:52:51,442:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function conditionally_calculate_md5 at 0x000002A91CD6C360>
2023-02-16 15:52:51,447:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function add_expect_header at 0x000002A91CF018A0>
2023-02-16 15:52:51,447:DEBUG:botocore.handlers:Adding expect 100 continue header to request.
2023-02-16 15:52:51,447:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function add_recursion_detection_header at 0x000002A91CEEFBA0>
2023-02-16 15:52:51,447:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function inject_api_version_header_if_needed at 0x000002A91CF02DE0>
2023-02-16 15:52:51,447:DEBUG:botocore.endpoint:Making request for OperationModel(name=PutObject) with params: {'url_path': '/6389b7ff41896d32c199d378.pdf', 'query_string': {}, 'method': 'PUT', 'headers': {'User-Agent': 'Boto3/1.26.71 Python/3.11.2 Windows/10 Botocore/1.29.71', 'Content-MD5': 'X6paoxeomp9WLBJDHXuEbA==', 'Expect': '100-continue'}, 'body': <s3transfer.utils.ReadFileChunk object at 0x000002A91D433150>, 'auth_path': '/arunpdf/6389b7ff41896d32c199d378.pdf', 'url': 'https://arunpdf.s3.ap-northeast-1.amazonaws.com/6389b7ff41896d32c199d378.pdf', 'context': {'client_region': 'ap-northeast-1', 'client_config': <botocore.config.Config object at 0x000002A91D43F750>, 'has_streaming_input': True, 'auth_type': 'v4', 'signing': {'region': 'ap-northeast-1', 'signing_name': 's3', 'disableDoubleEncoding': True}, 's3_redirect': {'redirected': False, 'bucket': 'arunpdf', 'params': {'Bucket': 'arunpdf', 'Key': '6389b7ff41896d32c199d378.pdf', 'Body': <s3transfer.utils.ReadFileChunk object at 0x000002A91D433150>}}}}
2023-02-16 15:52:51,447:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function signal_not_transferring at 0x000002A91DBB22A0>
2023-02-16 15:52:51,448:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <bound method RequestSigner.handler of <botocore.signers.RequestSigner object at 0x000002A91DC28550>>
2023-02-16 15:52:51,448:DEBUG:botocore.hooks:Event choose-signer.s3.PutObject: calling handler <bound method ClientCreator._default_s3_presign_to_sigv2 of <botocore.client.ClientCreator object at 0x000002A91D43FC50>>
2023-02-16 15:52:51,448:DEBUG:botocore.hooks:Event choose-signer.s3.PutObject: calling handler <function set_operation_specific_signer at 0x000002A91CF011C0>
2023-02-16 15:52:51,448:DEBUG:botocore.hooks:Event before-sign.s3.PutObject: calling handler <function remove_arn_from_signing_path at 0x000002A91CF03740>
2023-02-16 15:52:51,448:DEBUG:botocore.auth:Calculating signature using v4 auth.
2023-02-16 15:52:51,448:DEBUG:botocore.auth:CanonicalRequest:
PUT
/6389b7ff41896d32c199d378.pdf

content-md5:X6paoxeomp9WLBJDHXuEbA==
host:arunpdf.s3.ap-northeast-1.amazonaws.com
x-amz-content-sha256:UNSIGNED-PAYLOAD
x-amz-date:20230216T102251Z

content-md5;host;x-amz-content-sha256;x-amz-date
UNSIGNED-PAYLOAD
2023-02-16 15:52:51,448:DEBUG:botocore.auth:StringToSign:
AWS4-HMAC-SHA256
20230216T102251Z
20230216/ap-northeast-1/s3/aws4_request
263d814973c483a5ac2eadeede72130eef1d6c1633818b3160dbd4c4b37aebfe
2023-02-16 15:52:51,448:DEBUG:botocore.auth:Signature:
2016211bfebdbab29663f9df06c2d2334cc2e11ad910a31cf0573f0025e53cad
2023-02-16 15:52:51,448:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function signal_transferring at 0x000002A91DBB25C0>
2023-02-16 15:52:51,448:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function add_retry_headers at 0x000002A91CF03560>
2023-02-16 15:52:51,448:DEBUG:botocore.endpoint:Sending http request: <AWSPreparedRequest stream_output=False, method=PUT, url=https://arunpdf.s3.ap-northeast-1.amazonaws.com/6389b7ff41896d32c199d378.pdf, headers={'User-Agent': b'Boto3/1.26.71 Python/3.11.2 Windows/10 Botocore/1.29.71', 'Content-MD5': b'X6paoxeomp9WLBJDHXuEbA==', 'Expect': b'100-continue', 'X-Amz-Date': b'20230216T102251Z', 'X-Amz-Content-SHA256': b'UNSIGNED-PAYLOAD', 'Authorization': b'AWS4-HMAC-SHA256 Credential=AKIAZIH2ZW4NDK3AU2M6/20230216/ap-northeast-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=2016211bfebdbab29663f9df06c2d2334cc2e11ad910a31cf0573f0025e53cad', 'amz-sdk-invocation-id': b'1a2abb1c-532a-4a77-ab0d-9ca666e6e69b', 'amz-sdk-request': b'attempt=1', 'Content-Length': '2678'}>
2023-02-16 15:52:51,449:DEBUG:botocore.httpsession:Certificate path: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\cacert.pem
2023-02-16 15:52:51,449:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): arunpdf.s3.ap-northeast-1.amazonaws.com:443
2023-02-16 15:52:52,008:DEBUG:botocore.awsrequest:Waiting for 100 Continue response.
2023-02-16 15:52:52,194:DEBUG:botocore.awsrequest:100 Continue response seen, now sending request body.
2023-02-16 15:52:52,379:DEBUG:urllib3.connectionpool:https://arunpdf.s3.ap-northeast-1.amazonaws.com:443 "PUT /6389b7ff41896d32c199d378.pdf HTTP/1.1" 200 0
2023-02-16 15:52:52,380:DEBUG:botocore.parsers:Response headers: {'x-amz-id-2': 'VsvuYgtSl6Paz/Yun4C58J+9kqmuB+YIGqGNBl+P3xaco5SK0Ldg99Zj0u2B94AAJIAcDpFPKuo=', 'x-amz-request-id': 'R095H1SA18TB0Q5M', 'Date': 'Thu, 16 Feb 2023 10:22:54 GMT', 'x-amz-server-side-encryption': 'AES256', 'ETag': '"5faa5aa317a89a9f562c12431d7b846c"', 'Server': 'AmazonS3', 'Content-Length': '0'}
2023-02-16 15:52:52,380:DEBUG:botocore.parsers:Response body:
b''
2023-02-16 15:52:52,380:DEBUG:botocore.hooks:Event needs-retry.s3.PutObject: calling handler <botocore.retryhandler.RetryHandler object at 0x000002A91E0C3510>
2023-02-16 15:52:52,380:DEBUG:botocore.retryhandler:No retry needed.
2023-02-16 15:52:52,380:DEBUG:botocore.hooks:Event needs-retry.s3.PutObject: calling handler <bound method S3RegionRedirectorv2.redirect_from_error of <botocore.utils.S3RegionRedirectorv2 object at 0x000002A91E0B8050>>
2023-02-16 15:52:52,380:DEBUG:s3transfer.utils:Releasing acquire 0/None
2023-02-16 15:52:52,382:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:52:52] "POST /course_detail HTTP/1.1" 200 -
2023-02-16 15:53:55,176:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:53:55,178:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:53:55] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:53:57,793:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:53:57,794:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:53:57] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:53:57,876:INFO:root:MongoDB Connection...Done
2023-02-16 15:53:57,876:INFO:root:MongoDB Database...Done
2023-02-16 15:53:57,876:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 15:53:57,876:INFO:root:MongoDB Collection courses ...Created
2023-02-16 15:53:59,133:ERROR:root:batch op errors occurred, full error: {'writeErrors': [{'index': 0, 'code': 11000, 'keyPattern': {'_id': 1}, 'keyValue': {'_id': ObjectId('63ee0373dd2682e4965f4979')}, 'errmsg': "E11000 duplicate key error collection: ineuron.cat_subcat index: _id_ dup key: { _id: ObjectId('63ee0373dd2682e4965f4979') }", 'op': {'cat_id': '62cd958353f5fe61566b9ae4', 'cat_name': 'DATA SCIENCE', 'sub_cat_id': '62cd959d53f5fe14766b9ae5', 'sub_cat_name': 'FULL STACK DATA SCIENCE', '_id': ObjectId('63ee0373dd2682e4965f4979')}}], 'writeConcernErrors': [], 'nInserted': 0, 'nUpserted': 0, 'nMatched': 0, 'nModified': 0, 'nRemoved': 0, 'upserted': []}
2023-02-16 15:53:59,133:INFO:root:MongoDB Data Insert...Failed
2023-02-16 15:54:02,298:ERROR:root:batch op errors occurred, full error: {'writeErrors': [{'index': 0, 'code': 11000, 'keyPattern': {'_id': 1}, 'keyValue': {'_id': ObjectId('63ee0375dd2682e4965f4a2f')}, 'errmsg': "E11000 duplicate key error collection: ineuron.courses index: _id_ dup key: { _id: ObjectId('63ee0375dd2682e4965f4a2f') }", 'op': {'course_id': '63bbd0863d8c5c05632becbc', 'course_name': 'Complete DSA in Python', 'course_description': 'A comprehensive chase to excel any intreview for the Data Structures and Algorithms. This course has been specifically designed to provide resources that would assist you in cracking problem-solving interviews. The presented problems in the course would suffice to look on to positive outcomes in the interviews.', 'course_mode': 'self-paced', 'course_cat_id': '62cda11e53f5fe7a386b9b87', 'course_timings': {'timings': 'Flexible'}, 'course_curriculum': ['Introduction to Algorithms', 'Analysis in Algorithms', 'Array Data Structure', 'Heap Data Structure', 'Recursion', 'Divide and Conquer', 'Linked List Data Structure', 'Stack and Queue', 'Hashing Data Structure', 'Tree Data Structure', 'Binary Search Tree', 'Graph Traversal Algorithms', 'Application of greedy algorithm', 'Dynamic Programming', 'Research Area- P, NP, NP-Hard and NP-Complete Problems'], 'course_requirements': ['System with minimum i3 processor or better', 'At least 4 GB of RAM', 'Working internet connection', 'Dedication to learn'], 'course_features': ['Course Materials', 'Self Paced Learning', 'Lifetime Dashboard Access', 'Completion Certificate'], 'instructor_id': ['618964776a74763ac407df8b'], 'instructor_name': ['Priya Bhatia'], 'instructors_social': [{'linkedin': 'https://www.linkedin.com/in/bhatia-priya/', 'instagram': 'https://www.instagram.com/priyabhatia6971/', 'facebook': 'https://www.facebook.com/priya.bhatia.37604', 'github': 'https://github.com/priya6971'}], 'instructor_description': ['Expertise in data structure competitive programing and solving an analytical problems and implementing data structure algorithm in multiple programing language. I have done my M.Tech in Artificial Intelligence at IIT Hyderabad and have an experience of implementation in multiple projects.'], '_id': ObjectId('63ee0375dd2682e4965f4a2f')}}], 'writeConcernErrors': [], 'nInserted': 0, 'nUpserted': 0, 'nMatched': 0, 'nModified': 0, 'nRemoved': 0, 'upserted': []}
2023-02-16 15:54:02,298:INFO:root:MongoDB Data Insert...Failed
2023-02-16 15:54:02,988:INFO:root:Connected to MySQL Database...
2023-02-16 15:54:03,313:INFO:root:Created Database...iNeuronDb
2023-02-16 15:54:03,472:INFO:root:Using Database...iNeuronDb
2023-02-16 15:54:03,636:INFO:root:Created Table...courses
2023-02-16 15:54:03,636:INFO:root:Total data to Insert: 450
2023-02-16 15:55:20,105:INFO:root:Total data Inserted in Table: 1
2023-02-16 15:55:20,105:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 15:55:20,106:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:55:20] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 15:55:20,121:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:55:20,124:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:55:20] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:55:57,219:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:55:57,222:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:55:57] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:55:59,908:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:55:59,909:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:55:59] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:56:00,002:INFO:root:MongoDB Connection...Done
2023-02-16 15:56:00,002:INFO:root:MongoDB Database...Done
2023-02-16 15:56:00,002:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 15:56:00,002:INFO:root:MongoDB Collection courses ...Created
2023-02-16 15:56:01,243:ERROR:root:batch op errors occurred, full error: {'writeErrors': [{'index': 0, 'code': 11000, 'keyPattern': {'_id': 1}, 'keyValue': {'_id': ObjectId('63ee0373dd2682e4965f4979')}, 'errmsg': "E11000 duplicate key error collection: ineuron.cat_subcat index: _id_ dup key: { _id: ObjectId('63ee0373dd2682e4965f4979') }", 'op': {'cat_id': '62cd958353f5fe61566b9ae4', 'cat_name': 'DATA SCIENCE', 'sub_cat_id': '62cd959d53f5fe14766b9ae5', 'sub_cat_name': 'FULL STACK DATA SCIENCE', '_id': ObjectId('63ee0373dd2682e4965f4979')}}], 'writeConcernErrors': [], 'nInserted': 0, 'nUpserted': 0, 'nMatched': 0, 'nModified': 0, 'nRemoved': 0, 'upserted': []}
2023-02-16 15:56:01,243:INFO:root:MongoDB Data Insert...Failed
2023-02-16 15:56:03,244:ERROR:root:batch op errors occurred, full error: {'writeErrors': [{'index': 0, 'code': 11000, 'keyPattern': {'_id': 1}, 'keyValue': {'_id': ObjectId('63ee0375dd2682e4965f4a2f')}, 'errmsg': "E11000 duplicate key error collection: ineuron.courses index: _id_ dup key: { _id: ObjectId('63ee0375dd2682e4965f4a2f') }", 'op': {'course_id': '63bbd0863d8c5c05632becbc', 'course_name': 'Complete DSA in Python', 'course_description': 'A comprehensive chase to excel any intreview for the Data Structures and Algorithms. This course has been specifically designed to provide resources that would assist you in cracking problem-solving interviews. The presented problems in the course would suffice to look on to positive outcomes in the interviews.', 'course_mode': 'self-paced', 'course_cat_id': '62cda11e53f5fe7a386b9b87', 'course_timings': {'timings': 'Flexible'}, 'course_curriculum': ['Introduction to Algorithms', 'Analysis in Algorithms', 'Array Data Structure', 'Heap Data Structure', 'Recursion', 'Divide and Conquer', 'Linked List Data Structure', 'Stack and Queue', 'Hashing Data Structure', 'Tree Data Structure', 'Binary Search Tree', 'Graph Traversal Algorithms', 'Application of greedy algorithm', 'Dynamic Programming', 'Research Area- P, NP, NP-Hard and NP-Complete Problems'], 'course_requirements': ['System with minimum i3 processor or better', 'At least 4 GB of RAM', 'Working internet connection', 'Dedication to learn'], 'course_features': ['Course Materials', 'Self Paced Learning', 'Lifetime Dashboard Access', 'Completion Certificate'], 'instructor_id': ['618964776a74763ac407df8b'], 'instructor_name': ['Priya Bhatia'], 'instructors_social': [{'linkedin': 'https://www.linkedin.com/in/bhatia-priya/', 'instagram': 'https://www.instagram.com/priyabhatia6971/', 'facebook': 'https://www.facebook.com/priya.bhatia.37604', 'github': 'https://github.com/priya6971'}], 'instructor_description': ['Expertise in data structure competitive programing and solving an analytical problems and implementing data structure algorithm in multiple programing language. I have done my M.Tech in Artificial Intelligence at IIT Hyderabad and have an experience of implementation in multiple projects.'], '_id': ObjectId('63ee0375dd2682e4965f4a2f')}}], 'writeConcernErrors': [], 'nInserted': 0, 'nUpserted': 0, 'nMatched': 0, 'nModified': 0, 'nRemoved': 0, 'upserted': []}
2023-02-16 15:56:03,245:INFO:root:MongoDB Data Insert...Failed
2023-02-16 15:56:03,976:INFO:root:Connected to MySQL Database...
2023-02-16 15:56:04,296:INFO:root:Created Database...iNeuronDb
2023-02-16 15:56:04,452:INFO:root:Using Database...iNeuronDb
2023-02-16 15:56:04,639:INFO:root:Created Table...courses
2023-02-16 15:56:04,639:INFO:root:Total data to Insert: 450
2023-02-16 15:57:18,518:INFO:root:Total data Inserted in Table: 1
2023-02-16 15:57:18,519:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 15:57:18,519:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:57:18] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 15:57:18,532:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:57:18,536:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:57:18] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:57:27,313:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:57:27,315:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:57:27] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:57:29,129:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:57:29,131:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:57:29] "POST /course_list HTTP/1.1" 200 -
2023-02-16 15:57:43,487:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:57:43,489:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:57:43] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 15:59:35,297:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 15:59:35,548:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:35,548:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 15:59:37,881:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:37,882:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 15:59:37,893:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:37,893:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 15:59:37,893:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:37,893:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 15:59:37,917:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:37,917:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 15:59:37,917:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:37,917:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 15:59:37,920:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:37,920:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 15:59:37,922:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:37,922:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 15:59:37,958:DEBUG:NeuronScrapper:Done
2023-02-16 15:59:37,965:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 15:59:37,965:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 15:59:41,755:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:59:41,761:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:59:41] "GET / HTTP/1.1" 200 -
2023-02-16 15:59:43,717:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 15:59:43,719:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 15:59:43] "POST /home_page HTTP/1.1" 200 -
2023-02-16 15:59:43,804:INFO:root:MongoDB Connection...Done
2023-02-16 15:59:43,804:INFO:root:MongoDB Database...Done
2023-02-16 15:59:43,804:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 15:59:43,804:INFO:root:MongoDB Collection courses ...Created
2023-02-16 15:59:45,031:INFO:root:MongoDB Data Insert ...Done
2023-02-16 15:59:48,216:INFO:root:MongoDB Data Insert ...Done
2023-02-16 15:59:48,934:INFO:root:Connected to MySQL Database...
2023-02-16 15:59:49,323:INFO:root:Created Database...iNeuronDb
2023-02-16 15:59:49,490:INFO:root:Using Database...iNeuronDb
2023-02-16 15:59:49,654:INFO:root:Created Table...courses
2023-02-16 15:59:49,654:INFO:root:Total data to Insert: 450
2023-02-16 16:01:05,817:INFO:root:Total data Inserted in Table: 1
2023-02-16 16:01:05,817:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:01:05,818:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:05] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:01:05,832:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:05,839:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:05] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:01:09,297:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:09,301:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:09] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:01:11,280:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:11,290:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:11] "POST /course_list HTTP/1.1" 200 -
2023-02-16 16:01:13,862:INFO:root:File Name for PDF: 6392e14fe714c0a0c7531c96.pdf
2023-02-16 16:01:13,863:DEBUG:botocore.hooks:Changing event name from creating-client-class.iot-data to creating-client-class.iot-data-plane
2023-02-16 16:01:13,865:DEBUG:botocore.hooks:Changing event name from before-call.apigateway to before-call.api-gateway
2023-02-16 16:01:13,865:DEBUG:botocore.hooks:Changing event name from request-created.machinelearning.Predict to request-created.machine-learning.Predict
2023-02-16 16:01:13,866:DEBUG:botocore.hooks:Changing event name from before-parameter-build.autoscaling.CreateLaunchConfiguration to before-parameter-build.auto-scaling.CreateLaunchConfiguration
2023-02-16 16:01:13,866:DEBUG:botocore.hooks:Changing event name from before-parameter-build.route53 to before-parameter-build.route-53
2023-02-16 16:01:13,867:DEBUG:botocore.hooks:Changing event name from request-created.cloudsearchdomain.Search to request-created.cloudsearch-domain.Search
2023-02-16 16:01:13,867:DEBUG:botocore.hooks:Changing event name from docs.*.autoscaling.CreateLaunchConfiguration.complete-section to docs.*.auto-scaling.CreateLaunchConfiguration.complete-section
2023-02-16 16:01:13,869:DEBUG:botocore.hooks:Changing event name from before-parameter-build.logs.CreateExportTask to before-parameter-build.cloudwatch-logs.CreateExportTask
2023-02-16 16:01:13,869:DEBUG:botocore.hooks:Changing event name from docs.*.logs.CreateExportTask.complete-section to docs.*.cloudwatch-logs.CreateExportTask.complete-section
2023-02-16 16:01:13,870:DEBUG:botocore.hooks:Changing event name from before-parameter-build.cloudsearchdomain.Search to before-parameter-build.cloudsearch-domain.Search
2023-02-16 16:01:13,870:DEBUG:botocore.hooks:Changing event name from docs.*.cloudsearchdomain.Search.complete-section to docs.*.cloudsearch-domain.Search.complete-section
2023-02-16 16:01:13,871:DEBUG:botocore.utils:IMDS ENDPOINT: http://169.254.169.254/
2023-02-16 16:01:13,872:DEBUG:botocore.credentials:Looking for credentials via: env
2023-02-16 16:01:13,872:DEBUG:botocore.credentials:Looking for credentials via: assume-role
2023-02-16 16:01:13,872:DEBUG:botocore.credentials:Looking for credentials via: assume-role-with-web-identity
2023-02-16 16:01:13,872:DEBUG:botocore.credentials:Looking for credentials via: sso
2023-02-16 16:01:13,872:DEBUG:botocore.credentials:Looking for credentials via: shared-credentials-file
2023-02-16 16:01:13,873:INFO:botocore.credentials:Found credentials in shared credentials file: ~/.aws/credentials
2023-02-16 16:01:13,873:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\endpoints.json
2023-02-16 16:01:13,881:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\sdk-default-configuration.json
2023-02-16 16:01:13,881:DEBUG:botocore.hooks:Event choose-service-name: calling handler <function handle_service_name_alias at 0x000001E2480DFA60>
2023-02-16 16:01:13,918:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\s3\2006-03-01\service-2.json
2023-02-16 16:01:13,966:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\s3\2006-03-01\endpoint-rule-set-1.json.gz
2023-02-16 16:01:13,970:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\partitions.json
2023-02-16 16:01:13,971:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function add_generate_presigned_post at 0x000001E24801F880>
2023-02-16 16:01:13,971:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function lazy_call.<locals>._handler at 0x000001E2494E9300>
2023-02-16 16:01:13,990:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function add_generate_presigned_url at 0x000001E24801F600>
2023-02-16 16:01:13,991:DEBUG:botocore.endpoint:Setting s3 timeout as (60, 60)
2023-02-16 16:01:13,993:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\_retry.json
2023-02-16 16:01:13,993:DEBUG:botocore.client:Registering retry handlers for service: s3
2023-02-16 16:01:13,993:DEBUG:botocore.utils:Registering S3 region redirector handler
2023-02-16 16:01:13,994:DEBUG:s3transfer.utils:Acquiring 0
2023-02-16 16:01:13,994:DEBUG:s3transfer.tasks:UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000001E24927BA10>}) about to wait for the following futures []
2023-02-16 16:01:13,994:DEBUG:s3transfer.tasks:UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000001E24927BA10>}) done waiting for dependent futures
2023-02-16 16:01:13,994:DEBUG:s3transfer.tasks:Executing task UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000001E24927BA10>}) with kwargs {'client': <botocore.client.S3 object at 0x000001E248D62A50>, 'config': <boto3.s3.transfer.TransferConfig object at 0x000001E2485E0290>, 'osutil': <s3transfer.utils.OSUtils object at 0x000001E24925EF90>, 'request_executor': <s3transfer.futures.BoundedExecutor object at 0x000001E248E3C450>, 'transfer_future': <s3transfer.futures.TransferFuture object at 0x000001E24927BA10>}
2023-02-16 16:01:13,994:DEBUG:s3transfer.futures:Submitting task PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '6392e14fe714c0a0c7531c96.pdf', 'extra_args': {}}) to executor <s3transfer.futures.BoundedExecutor object at 0x000001E248E3C450> for transfer request: 0.
2023-02-16 16:01:13,994:DEBUG:s3transfer.utils:Acquiring 0
2023-02-16 16:01:13,994:DEBUG:s3transfer.tasks:PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '6392e14fe714c0a0c7531c96.pdf', 'extra_args': {}}) about to wait for the following futures []
2023-02-16 16:01:13,995:DEBUG:s3transfer.utils:Releasing acquire 0/None
2023-02-16 16:01:13,995:DEBUG:s3transfer.tasks:PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '6392e14fe714c0a0c7531c96.pdf', 'extra_args': {}}) done waiting for dependent futures
2023-02-16 16:01:13,995:DEBUG:s3transfer.tasks:Executing task PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '6392e14fe714c0a0c7531c96.pdf', 'extra_args': {}}) with kwargs {'client': <botocore.client.S3 object at 0x000001E248D62A50>, 'fileobj': <s3transfer.utils.ReadFileChunk object at 0x000001E2492BFC50>, 'bucket': 'arunpdf', 'key': '6392e14fe714c0a0c7531c96.pdf', 'extra_args': {}}
2023-02-16 16:01:13,995:DEBUG:botocore.hooks:Event before-endpoint-resolution.s3: calling handler <function customize_endpoint_resolver_builtins at 0x000001E2480F37E0>
2023-02-16 16:01:13,995:DEBUG:botocore.hooks:Event before-endpoint-resolution.s3: calling handler <bound method S3RegionRedirectorv2.redirect_from_cache of <botocore.utils.S3RegionRedirectorv2 object at 0x000001E248E33910>>
2023-02-16 16:01:13,996:DEBUG:botocore.regions:Calling endpoint provider with parameters: {'Bucket': 'arunpdf', 'Region': 'ap-northeast-1', 'UseFIPS': False, 'UseDualStack': False, 'ForcePathStyle': False, 'Accelerate': False, 'UseGlobalEndpoint': False, 'DisableMultiRegionAccessPoints': False, 'UseArnRegion': True}
2023-02-16 16:01:13,996:DEBUG:botocore.regions:Endpoint provider result: https://arunpdf.s3.ap-northeast-1.amazonaws.com
2023-02-16 16:01:13,996:DEBUG:botocore.regions:Selecting from endpoint provider's list of auth schemes: "sigv4". User selected auth scheme is: "None"
2023-02-16 16:01:13,996:DEBUG:botocore.regions:Selected auth type "v4" as "v4" with signing context params: {'region': 'ap-northeast-1', 'signing_name': 's3', 'disableDoubleEncoding': True}
2023-02-16 16:01:13,997:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function validate_ascii_metadata at 0x000001E2480F22A0>
2023-02-16 16:01:13,997:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function sse_md5 at 0x000001E2480F1580>
2023-02-16 16:01:13,997:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function convert_body_to_file_like_object at 0x000001E2480F2CA0>
2023-02-16 16:01:13,997:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function validate_bucket_name at 0x000001E2480F14E0>
2023-02-16 16:01:13,997:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function remove_bucket_from_url_paths_from_model at 0x000001E2480F3600>
2023-02-16 16:01:13,997:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <bound method S3RegionRedirectorv2.annotate_request_context of <botocore.utils.S3RegionRedirectorv2 object at 0x000001E248E33910>>
2023-02-16 16:01:13,997:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function generate_idempotent_uuid at 0x000001E2480F1300>
2023-02-16 16:01:13,997:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function conditionally_calculate_md5 at 0x000001E247F5C360>
2023-02-16 16:01:14,002:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function add_expect_header at 0x000001E2480F18A0>
2023-02-16 16:01:14,002:DEBUG:botocore.handlers:Adding expect 100 continue header to request.
2023-02-16 16:01:14,002:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function add_recursion_detection_header at 0x000001E2480DFBA0>
2023-02-16 16:01:14,002:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function inject_api_version_header_if_needed at 0x000001E2480F2DE0>
2023-02-16 16:01:14,002:DEBUG:botocore.endpoint:Making request for OperationModel(name=PutObject) with params: {'url_path': '/6392e14fe714c0a0c7531c96.pdf', 'query_string': {}, 'method': 'PUT', 'headers': {'User-Agent': 'Boto3/1.26.71 Python/3.11.2 Windows/10 Botocore/1.29.71', 'Content-MD5': 'Cq8ZRZ1nTWEjkcYiIgHmqA==', 'Expect': '100-continue'}, 'body': <s3transfer.utils.ReadFileChunk object at 0x000001E2492BFC50>, 'auth_path': '/arunpdf/6392e14fe714c0a0c7531c96.pdf', 'url': 'https://arunpdf.s3.ap-northeast-1.amazonaws.com/6392e14fe714c0a0c7531c96.pdf', 'context': {'client_region': 'ap-northeast-1', 'client_config': <botocore.config.Config object at 0x000001E24928D210>, 'has_streaming_input': True, 'auth_type': 'v4', 'signing': {'region': 'ap-northeast-1', 'signing_name': 's3', 'disableDoubleEncoding': True}, 's3_redirect': {'redirected': False, 'bucket': 'arunpdf', 'params': {'Bucket': 'arunpdf', 'Key': '6392e14fe714c0a0c7531c96.pdf', 'Body': <s3transfer.utils.ReadFileChunk object at 0x000001E2492BFC50>}}}}
2023-02-16 16:01:14,003:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function signal_not_transferring at 0x000001E248DA2200>
2023-02-16 16:01:14,003:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <bound method RequestSigner.handler of <botocore.signers.RequestSigner object at 0x000001E248E175D0>>
2023-02-16 16:01:14,003:DEBUG:botocore.hooks:Event choose-signer.s3.PutObject: calling handler <bound method ClientCreator._default_s3_presign_to_sigv2 of <botocore.client.ClientCreator object at 0x000001E2498A6A50>>
2023-02-16 16:01:14,003:DEBUG:botocore.hooks:Event choose-signer.s3.PutObject: calling handler <function set_operation_specific_signer at 0x000001E2480F11C0>
2023-02-16 16:01:14,003:DEBUG:botocore.hooks:Event before-sign.s3.PutObject: calling handler <function remove_arn_from_signing_path at 0x000001E2480F3740>
2023-02-16 16:01:14,003:DEBUG:botocore.auth:Calculating signature using v4 auth.
2023-02-16 16:01:14,003:DEBUG:botocore.auth:CanonicalRequest:
PUT
/6392e14fe714c0a0c7531c96.pdf

content-md5:Cq8ZRZ1nTWEjkcYiIgHmqA==
host:arunpdf.s3.ap-northeast-1.amazonaws.com
x-amz-content-sha256:UNSIGNED-PAYLOAD
x-amz-date:20230216T103114Z

content-md5;host;x-amz-content-sha256;x-amz-date
UNSIGNED-PAYLOAD
2023-02-16 16:01:14,003:DEBUG:botocore.auth:StringToSign:
AWS4-HMAC-SHA256
20230216T103114Z
20230216/ap-northeast-1/s3/aws4_request
f81ec94450a467a908df638cec2da8dd0c127371b2a5577559c8cf9047a08826
2023-02-16 16:01:14,004:DEBUG:botocore.auth:Signature:
09614aa079ebba91406042cc597182ae3f3f332a77518e854783e4b8adcfa34d
2023-02-16 16:01:14,004:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function signal_transferring at 0x000001E248DA2520>
2023-02-16 16:01:14,004:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function add_retry_headers at 0x000001E2480F3560>
2023-02-16 16:01:14,004:DEBUG:botocore.endpoint:Sending http request: <AWSPreparedRequest stream_output=False, method=PUT, url=https://arunpdf.s3.ap-northeast-1.amazonaws.com/6392e14fe714c0a0c7531c96.pdf, headers={'User-Agent': b'Boto3/1.26.71 Python/3.11.2 Windows/10 Botocore/1.29.71', 'Content-MD5': b'Cq8ZRZ1nTWEjkcYiIgHmqA==', 'Expect': b'100-continue', 'X-Amz-Date': b'20230216T103114Z', 'X-Amz-Content-SHA256': b'UNSIGNED-PAYLOAD', 'Authorization': b'AWS4-HMAC-SHA256 Credential=AKIAZIH2ZW4NDK3AU2M6/20230216/ap-northeast-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=09614aa079ebba91406042cc597182ae3f3f332a77518e854783e4b8adcfa34d', 'amz-sdk-invocation-id': b'd805b211-15dc-43af-9bdf-3908dd13eabc', 'amz-sdk-request': b'attempt=1', 'Content-Length': '2222'}>
2023-02-16 16:01:14,004:DEBUG:botocore.httpsession:Certificate path: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\cacert.pem
2023-02-16 16:01:14,004:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): arunpdf.s3.ap-northeast-1.amazonaws.com:443
2023-02-16 16:01:14,566:DEBUG:botocore.awsrequest:Waiting for 100 Continue response.
2023-02-16 16:01:14,759:DEBUG:botocore.awsrequest:100 Continue response seen, now sending request body.
2023-02-16 16:01:14,941:DEBUG:urllib3.connectionpool:https://arunpdf.s3.ap-northeast-1.amazonaws.com:443 "PUT /6392e14fe714c0a0c7531c96.pdf HTTP/1.1" 200 0
2023-02-16 16:01:14,942:DEBUG:botocore.parsers:Response headers: {'x-amz-id-2': 'aW2nOYGuwZZqBz/9ZkjbmUl3xN0e6dp8eJNtN+DeGj2IwsW7NY0v+7GfIPXbKxNF1b4AyUzixhQ=', 'x-amz-request-id': '3T5R6P0RHX45CYWM', 'Date': 'Thu, 16 Feb 2023 10:31:16 GMT', 'x-amz-server-side-encryption': 'AES256', 'ETag': '"0aaf19459d674d612391c6222201e6a8"', 'Server': 'AmazonS3', 'Content-Length': '0'}
2023-02-16 16:01:14,942:DEBUG:botocore.parsers:Response body:
b''
2023-02-16 16:01:14,942:DEBUG:botocore.hooks:Event needs-retry.s3.PutObject: calling handler <botocore.retryhandler.RetryHandler object at 0x000001E2492A3850>
2023-02-16 16:01:14,942:DEBUG:botocore.retryhandler:No retry needed.
2023-02-16 16:01:14,942:DEBUG:botocore.hooks:Event needs-retry.s3.PutObject: calling handler <bound method S3RegionRedirectorv2.redirect_from_error of <botocore.utils.S3RegionRedirectorv2 object at 0x000001E248E33910>>
2023-02-16 16:01:14,942:DEBUG:s3transfer.utils:Releasing acquire 0/None
2023-02-16 16:01:14,944:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:14] "POST /course_detail HTTP/1.1" 200 -
2023-02-16 16:01:21,695:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:21,696:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:21] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:01:23,238:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:23,241:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:23] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:01:27,002:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:27,004:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:27] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:01:28,503:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:28,504:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:28] "POST /course_list HTTP/1.1" 200 -
2023-02-16 16:01:32,223:INFO:root:File Name for PDF: 62f6868a4a19f1e052ebad5f.pdf
2023-02-16 16:01:32,224:DEBUG:botocore.hooks:Event choose-service-name: calling handler <function handle_service_name_alias at 0x000001E2480DFA60>
2023-02-16 16:01:32,225:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function add_generate_presigned_post at 0x000001E24801F880>
2023-02-16 16:01:32,225:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function lazy_call.<locals>._handler at 0x000001E2494E9300>
2023-02-16 16:01:32,225:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function add_generate_presigned_url at 0x000001E24801F600>
2023-02-16 16:01:32,226:DEBUG:botocore.endpoint:Setting s3 timeout as (60, 60)
2023-02-16 16:01:32,227:DEBUG:botocore.client:Registering retry handlers for service: s3
2023-02-16 16:01:32,227:DEBUG:botocore.utils:Registering S3 region redirector handler
2023-02-16 16:01:32,228:DEBUG:s3transfer.utils:Acquiring 0
2023-02-16 16:01:32,228:DEBUG:s3transfer.tasks:UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000001E248E52CD0>}) about to wait for the following futures []
2023-02-16 16:01:32,228:DEBUG:s3transfer.tasks:UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000001E248E52CD0>}) done waiting for dependent futures
2023-02-16 16:01:32,229:DEBUG:s3transfer.tasks:Executing task UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x000001E248E52CD0>}) with kwargs {'client': <botocore.client.S3 object at 0x000001E248E3C9D0>, 'config': <boto3.s3.transfer.TransferConfig object at 0x000001E248E079D0>, 'osutil': <s3transfer.utils.OSUtils object at 0x000001E248E3DE90>, 'request_executor': <s3transfer.futures.BoundedExecutor object at 0x000001E248E52FD0>, 'transfer_future': <s3transfer.futures.TransferFuture object at 0x000001E248E52CD0>}
2023-02-16 16:01:32,229:DEBUG:s3transfer.futures:Submitting task PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '62f6868a4a19f1e052ebad5f.pdf', 'extra_args': {}}) to executor <s3transfer.futures.BoundedExecutor object at 0x000001E248E52FD0> for transfer request: 0.
2023-02-16 16:01:32,229:DEBUG:s3transfer.utils:Acquiring 0
2023-02-16 16:01:32,229:DEBUG:s3transfer.tasks:PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '62f6868a4a19f1e052ebad5f.pdf', 'extra_args': {}}) about to wait for the following futures []
2023-02-16 16:01:32,229:DEBUG:s3transfer.utils:Releasing acquire 0/None
2023-02-16 16:01:32,229:DEBUG:s3transfer.tasks:PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '62f6868a4a19f1e052ebad5f.pdf', 'extra_args': {}}) done waiting for dependent futures
2023-02-16 16:01:32,230:DEBUG:s3transfer.tasks:Executing task PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '62f6868a4a19f1e052ebad5f.pdf', 'extra_args': {}}) with kwargs {'client': <botocore.client.S3 object at 0x000001E248E3C9D0>, 'fileobj': <s3transfer.utils.ReadFileChunk object at 0x000001E248E3DC90>, 'bucket': 'arunpdf', 'key': '62f6868a4a19f1e052ebad5f.pdf', 'extra_args': {}}
2023-02-16 16:01:32,230:DEBUG:botocore.hooks:Event before-endpoint-resolution.s3: calling handler <function customize_endpoint_resolver_builtins at 0x000001E2480F37E0>
2023-02-16 16:01:32,230:DEBUG:botocore.hooks:Event before-endpoint-resolution.s3: calling handler <bound method S3RegionRedirectorv2.redirect_from_cache of <botocore.utils.S3RegionRedirectorv2 object at 0x000001E248E77F90>>
2023-02-16 16:01:32,230:DEBUG:botocore.regions:Calling endpoint provider with parameters: {'Bucket': 'arunpdf', 'Region': 'ap-northeast-1', 'UseFIPS': False, 'UseDualStack': False, 'ForcePathStyle': False, 'Accelerate': False, 'UseGlobalEndpoint': False, 'DisableMultiRegionAccessPoints': False, 'UseArnRegion': True}
2023-02-16 16:01:32,230:DEBUG:botocore.regions:Endpoint provider result: https://arunpdf.s3.ap-northeast-1.amazonaws.com
2023-02-16 16:01:32,230:DEBUG:botocore.regions:Selecting from endpoint provider's list of auth schemes: "sigv4". User selected auth scheme is: "None"
2023-02-16 16:01:32,230:DEBUG:botocore.regions:Selected auth type "v4" as "v4" with signing context params: {'region': 'ap-northeast-1', 'signing_name': 's3', 'disableDoubleEncoding': True}
2023-02-16 16:01:32,231:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function validate_ascii_metadata at 0x000001E2480F22A0>
2023-02-16 16:01:32,231:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function sse_md5 at 0x000001E2480F1580>
2023-02-16 16:01:32,231:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function convert_body_to_file_like_object at 0x000001E2480F2CA0>
2023-02-16 16:01:32,231:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function validate_bucket_name at 0x000001E2480F14E0>
2023-02-16 16:01:32,231:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function remove_bucket_from_url_paths_from_model at 0x000001E2480F3600>
2023-02-16 16:01:32,231:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <bound method S3RegionRedirectorv2.annotate_request_context of <botocore.utils.S3RegionRedirectorv2 object at 0x000001E248E77F90>>
2023-02-16 16:01:32,231:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function generate_idempotent_uuid at 0x000001E2480F1300>
2023-02-16 16:01:32,231:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function conditionally_calculate_md5 at 0x000001E247F5C360>
2023-02-16 16:01:32,236:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function add_expect_header at 0x000001E2480F18A0>
2023-02-16 16:01:32,236:DEBUG:botocore.handlers:Adding expect 100 continue header to request.
2023-02-16 16:01:32,236:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function add_recursion_detection_header at 0x000001E2480DFBA0>
2023-02-16 16:01:32,236:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function inject_api_version_header_if_needed at 0x000001E2480F2DE0>
2023-02-16 16:01:32,236:DEBUG:botocore.endpoint:Making request for OperationModel(name=PutObject) with params: {'url_path': '/62f6868a4a19f1e052ebad5f.pdf', 'query_string': {}, 'method': 'PUT', 'headers': {'User-Agent': 'Boto3/1.26.71 Python/3.11.2 Windows/10 Botocore/1.29.71', 'Content-MD5': 'ZmEUEgyeJKeSvi64BCrcuA==', 'Expect': '100-continue'}, 'body': <s3transfer.utils.ReadFileChunk object at 0x000001E248E3DC90>, 'auth_path': '/arunpdf/62f6868a4a19f1e052ebad5f.pdf', 'url': 'https://arunpdf.s3.ap-northeast-1.amazonaws.com/62f6868a4a19f1e052ebad5f.pdf', 'context': {'client_region': 'ap-northeast-1', 'client_config': <botocore.config.Config object at 0x000001E248E770D0>, 'has_streaming_input': True, 'auth_type': 'v4', 'signing': {'region': 'ap-northeast-1', 'signing_name': 's3', 'disableDoubleEncoding': True}, 's3_redirect': {'redirected': False, 'bucket': 'arunpdf', 'params': {'Bucket': 'arunpdf', 'Key': '62f6868a4a19f1e052ebad5f.pdf', 'Body': <s3transfer.utils.ReadFileChunk object at 0x000001E248E3DC90>}}}}
2023-02-16 16:01:32,236:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function signal_not_transferring at 0x000001E248DA2200>
2023-02-16 16:01:32,236:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <bound method RequestSigner.handler of <botocore.signers.RequestSigner object at 0x000001E248E77090>>
2023-02-16 16:01:32,236:DEBUG:botocore.hooks:Event choose-signer.s3.PutObject: calling handler <bound method ClientCreator._default_s3_presign_to_sigv2 of <botocore.client.ClientCreator object at 0x000001E248E075D0>>
2023-02-16 16:01:32,236:DEBUG:botocore.hooks:Event choose-signer.s3.PutObject: calling handler <function set_operation_specific_signer at 0x000001E2480F11C0>
2023-02-16 16:01:32,237:DEBUG:botocore.hooks:Event before-sign.s3.PutObject: calling handler <function remove_arn_from_signing_path at 0x000001E2480F3740>
2023-02-16 16:01:32,237:DEBUG:botocore.auth:Calculating signature using v4 auth.
2023-02-16 16:01:32,237:DEBUG:botocore.auth:CanonicalRequest:
PUT
/62f6868a4a19f1e052ebad5f.pdf

content-md5:ZmEUEgyeJKeSvi64BCrcuA==
host:arunpdf.s3.ap-northeast-1.amazonaws.com
x-amz-content-sha256:UNSIGNED-PAYLOAD
x-amz-date:20230216T103132Z

content-md5;host;x-amz-content-sha256;x-amz-date
UNSIGNED-PAYLOAD
2023-02-16 16:01:32,237:DEBUG:botocore.auth:StringToSign:
AWS4-HMAC-SHA256
20230216T103132Z
20230216/ap-northeast-1/s3/aws4_request
e252c147385e460f3b99dd44def3c9b56140be88da658d913267e76ca3064fde
2023-02-16 16:01:32,237:DEBUG:botocore.auth:Signature:
8a7fdd46f895c19a8e5e40196b82d4568198142e0179923d9cc66623b109ceea
2023-02-16 16:01:32,237:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function signal_transferring at 0x000001E248DA2520>
2023-02-16 16:01:32,237:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function add_retry_headers at 0x000001E2480F3560>
2023-02-16 16:01:32,237:DEBUG:botocore.endpoint:Sending http request: <AWSPreparedRequest stream_output=False, method=PUT, url=https://arunpdf.s3.ap-northeast-1.amazonaws.com/62f6868a4a19f1e052ebad5f.pdf, headers={'User-Agent': b'Boto3/1.26.71 Python/3.11.2 Windows/10 Botocore/1.29.71', 'Content-MD5': b'ZmEUEgyeJKeSvi64BCrcuA==', 'Expect': b'100-continue', 'X-Amz-Date': b'20230216T103132Z', 'X-Amz-Content-SHA256': b'UNSIGNED-PAYLOAD', 'Authorization': b'AWS4-HMAC-SHA256 Credential=AKIAZIH2ZW4NDK3AU2M6/20230216/ap-northeast-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=8a7fdd46f895c19a8e5e40196b82d4568198142e0179923d9cc66623b109ceea', 'amz-sdk-invocation-id': b'18db786e-6230-43b7-bf05-7c7ebd8a9faf', 'amz-sdk-request': b'attempt=1', 'Content-Length': '2274'}>
2023-02-16 16:01:32,237:DEBUG:botocore.httpsession:Certificate path: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\cacert.pem
2023-02-16 16:01:32,238:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): arunpdf.s3.ap-northeast-1.amazonaws.com:443
2023-02-16 16:01:32,792:DEBUG:botocore.awsrequest:Waiting for 100 Continue response.
2023-02-16 16:01:32,979:DEBUG:botocore.awsrequest:100 Continue response seen, now sending request body.
2023-02-16 16:01:33,163:DEBUG:urllib3.connectionpool:https://arunpdf.s3.ap-northeast-1.amazonaws.com:443 "PUT /62f6868a4a19f1e052ebad5f.pdf HTTP/1.1" 200 0
2023-02-16 16:01:33,163:DEBUG:botocore.parsers:Response headers: {'x-amz-id-2': 'J6I/HLxtndiMT6rsh6rJ7OL3Y2FyuKhdGFjW9YO3smr32CLWSjGheIL/1L79CNfKlkBkCuAwJs4=', 'x-amz-request-id': '66Y3Y9D5MJ5K26B9', 'Date': 'Thu, 16 Feb 2023 10:31:35 GMT', 'x-amz-server-side-encryption': 'AES256', 'ETag': '"666114120c9e24a792be2eb8042adcb8"', 'Server': 'AmazonS3', 'Content-Length': '0'}
2023-02-16 16:01:33,163:DEBUG:botocore.parsers:Response body:
b''
2023-02-16 16:01:33,164:DEBUG:botocore.hooks:Event needs-retry.s3.PutObject: calling handler <botocore.retryhandler.RetryHandler object at 0x000001E248E8F3D0>
2023-02-16 16:01:33,164:DEBUG:botocore.retryhandler:No retry needed.
2023-02-16 16:01:33,164:DEBUG:botocore.hooks:Event needs-retry.s3.PutObject: calling handler <bound method S3RegionRedirectorv2.redirect_from_error of <botocore.utils.S3RegionRedirectorv2 object at 0x000001E248E77F90>>
2023-02-16 16:01:33,164:DEBUG:s3transfer.utils:Releasing acquire 0/None
2023-02-16 16:01:33,166:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:33] "POST /course_detail HTTP/1.1" 200 -
2023-02-16 16:01:35,942:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:35,944:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:35] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:01:36,895:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:36,899:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:36] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:01:41,729:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:01:41,730:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:01:41] "GET /home_page HTTP/1.1" 200 -
2023-02-16 16:15:57,852:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 16:15:58,216:DEBUG:NeuronScrapper:Done
2023-02-16 16:15:58,216:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 16:16:00,499:DEBUG:NeuronScrapper:Done
2023-02-16 16:16:00,499:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 16:16:00,499:DEBUG:NeuronScrapper:Done
2023-02-16 16:16:00,499:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 16:16:00,499:DEBUG:NeuronScrapper:Done
2023-02-16 16:16:00,499:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 16:16:00,530:DEBUG:NeuronScrapper:Done
2023-02-16 16:16:00,530:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 16:16:00,530:DEBUG:NeuronScrapper:Done
2023-02-16 16:16:00,530:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 16:16:00,530:DEBUG:NeuronScrapper:Done
2023-02-16 16:16:00,530:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 16:16:00,530:DEBUG:NeuronScrapper:Done
2023-02-16 16:16:00,530:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 16:16:00,577:DEBUG:NeuronScrapper:Done
2023-02-16 16:16:00,577:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 16:16:00,577:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 16:16:10,879:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:16:10,885:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:16:10] "GET / HTTP/1.1" 200 -
2023-02-16 16:16:14,130:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:16:14,132:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:16:14] "POST /home_page HTTP/1.1" 200 -
2023-02-16 16:16:14,230:INFO:root:MongoDB Connection...Done
2023-02-16 16:16:14,230:INFO:root:MongoDB Database...Done
2023-02-16 16:16:14,230:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 16:16:14,230:INFO:root:MongoDB Collection courses ...Created
2023-02-16 16:16:15,534:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:16:17,565:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:16:18,310:INFO:root:Connected to MySQL Database...
2023-02-16 16:16:18,669:INFO:root:Created Database...iNeuronDb
2023-02-16 16:16:18,856:INFO:root:Using Database...iNeuronDb
2023-02-16 16:16:19,046:INFO:root:Created Table...courses
2023-02-16 16:16:19,046:INFO:root:Total data to Insert: 450
2023-02-16 16:17:45,206:INFO:root:Total data Inserted in Table: 1
2023-02-16 16:17:45,206:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:17:45,206:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:17:45] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:17:45,224:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:17:45,229:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:17:45] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:18:05,372:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:18:05,376:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:18:05] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:18:07,009:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:18:07,018:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:18:07] "POST /course_list HTTP/1.1" 200 -
2023-02-16 16:18:11,042:INFO:root:File Name for PDF: 62d6d281ff20c4b7c05d1e37.pdf
2023-02-16 16:18:11,043:DEBUG:botocore.hooks:Changing event name from creating-client-class.iot-data to creating-client-class.iot-data-plane
2023-02-16 16:18:11,045:DEBUG:botocore.hooks:Changing event name from before-call.apigateway to before-call.api-gateway
2023-02-16 16:18:11,045:DEBUG:botocore.hooks:Changing event name from request-created.machinelearning.Predict to request-created.machine-learning.Predict
2023-02-16 16:18:11,046:DEBUG:botocore.hooks:Changing event name from before-parameter-build.autoscaling.CreateLaunchConfiguration to before-parameter-build.auto-scaling.CreateLaunchConfiguration
2023-02-16 16:18:11,046:DEBUG:botocore.hooks:Changing event name from before-parameter-build.route53 to before-parameter-build.route-53
2023-02-16 16:18:11,047:DEBUG:botocore.hooks:Changing event name from request-created.cloudsearchdomain.Search to request-created.cloudsearch-domain.Search
2023-02-16 16:18:11,047:DEBUG:botocore.hooks:Changing event name from docs.*.autoscaling.CreateLaunchConfiguration.complete-section to docs.*.auto-scaling.CreateLaunchConfiguration.complete-section
2023-02-16 16:18:11,049:DEBUG:botocore.hooks:Changing event name from before-parameter-build.logs.CreateExportTask to before-parameter-build.cloudwatch-logs.CreateExportTask
2023-02-16 16:18:11,049:DEBUG:botocore.hooks:Changing event name from docs.*.logs.CreateExportTask.complete-section to docs.*.cloudwatch-logs.CreateExportTask.complete-section
2023-02-16 16:18:11,049:DEBUG:botocore.hooks:Changing event name from before-parameter-build.cloudsearchdomain.Search to before-parameter-build.cloudsearch-domain.Search
2023-02-16 16:18:11,050:DEBUG:botocore.hooks:Changing event name from docs.*.cloudsearchdomain.Search.complete-section to docs.*.cloudsearch-domain.Search.complete-section
2023-02-16 16:18:11,051:DEBUG:botocore.utils:IMDS ENDPOINT: http://169.254.169.254/
2023-02-16 16:18:11,052:DEBUG:botocore.credentials:Looking for credentials via: env
2023-02-16 16:18:11,052:DEBUG:botocore.credentials:Looking for credentials via: assume-role
2023-02-16 16:18:11,052:DEBUG:botocore.credentials:Looking for credentials via: assume-role-with-web-identity
2023-02-16 16:18:11,052:DEBUG:botocore.credentials:Looking for credentials via: sso
2023-02-16 16:18:11,052:DEBUG:botocore.credentials:Looking for credentials via: shared-credentials-file
2023-02-16 16:18:11,053:INFO:botocore.credentials:Found credentials in shared credentials file: ~/.aws/credentials
2023-02-16 16:18:11,053:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\endpoints.json
2023-02-16 16:18:11,060:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\sdk-default-configuration.json
2023-02-16 16:18:11,061:DEBUG:botocore.hooks:Event choose-service-name: calling handler <function handle_service_name_alias at 0x00000204ABDEFA60>
2023-02-16 16:18:11,093:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\s3\2006-03-01\service-2.json
2023-02-16 16:18:11,139:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\s3\2006-03-01\endpoint-rule-set-1.json.gz
2023-02-16 16:18:11,143:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\partitions.json
2023-02-16 16:18:11,145:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function add_generate_presigned_post at 0x00000204ABD2F880>
2023-02-16 16:18:11,145:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function lazy_call.<locals>._handler at 0x00000204AD208CC0>
2023-02-16 16:18:11,163:DEBUG:botocore.hooks:Event creating-client-class.s3: calling handler <function add_generate_presigned_url at 0x00000204ABD2F600>
2023-02-16 16:18:11,164:DEBUG:botocore.endpoint:Setting s3 timeout as (60, 60)
2023-02-16 16:18:11,165:DEBUG:botocore.loaders:Loading JSON file: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\data\_retry.json
2023-02-16 16:18:11,166:DEBUG:botocore.client:Registering retry handlers for service: s3
2023-02-16 16:18:11,166:DEBUG:botocore.utils:Registering S3 region redirector handler
2023-02-16 16:18:11,166:DEBUG:s3transfer.utils:Acquiring 0
2023-02-16 16:18:11,167:DEBUG:s3transfer.tasks:UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x00000204AC7397D0>}) about to wait for the following futures []
2023-02-16 16:18:11,167:DEBUG:s3transfer.tasks:UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x00000204AC7397D0>}) done waiting for dependent futures
2023-02-16 16:18:11,167:DEBUG:s3transfer.tasks:Executing task UploadSubmissionTask(transfer_id=0, {'transfer_future': <s3transfer.futures.TransferFuture object at 0x00000204AC7397D0>}) with kwargs {'client': <botocore.client.S3 object at 0x00000204AD5D9010>, 'config': <boto3.s3.transfer.TransferConfig object at 0x00000204ACFC9B50>, 'osutil': <s3transfer.utils.OSUtils object at 0x00000204AC31E5D0>, 'request_executor': <s3transfer.futures.BoundedExecutor object at 0x00000204ACFB2610>, 'transfer_future': <s3transfer.futures.TransferFuture object at 0x00000204AC7397D0>}
2023-02-16 16:18:11,168:DEBUG:s3transfer.futures:Submitting task PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '62d6d281ff20c4b7c05d1e37.pdf', 'extra_args': {}}) to executor <s3transfer.futures.BoundedExecutor object at 0x00000204ACFB2610> for transfer request: 0.
2023-02-16 16:18:11,168:DEBUG:s3transfer.utils:Acquiring 0
2023-02-16 16:18:11,168:DEBUG:s3transfer.tasks:PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '62d6d281ff20c4b7c05d1e37.pdf', 'extra_args': {}}) about to wait for the following futures []
2023-02-16 16:18:11,168:DEBUG:s3transfer.utils:Releasing acquire 0/None
2023-02-16 16:18:11,168:DEBUG:s3transfer.tasks:PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '62d6d281ff20c4b7c05d1e37.pdf', 'extra_args': {}}) done waiting for dependent futures
2023-02-16 16:18:11,168:DEBUG:s3transfer.tasks:Executing task PutObjectTask(transfer_id=0, {'bucket': 'arunpdf', 'key': '62d6d281ff20c4b7c05d1e37.pdf', 'extra_args': {}}) with kwargs {'client': <botocore.client.S3 object at 0x00000204AD5D9010>, 'fileobj': <s3transfer.utils.ReadFileChunk object at 0x00000204ACFD24D0>, 'bucket': 'arunpdf', 'key': '62d6d281ff20c4b7c05d1e37.pdf', 'extra_args': {}}
2023-02-16 16:18:11,168:DEBUG:botocore.hooks:Event before-endpoint-resolution.s3: calling handler <function customize_endpoint_resolver_builtins at 0x00000204ABE037E0>
2023-02-16 16:18:11,168:DEBUG:botocore.hooks:Event before-endpoint-resolution.s3: calling handler <bound method S3RegionRedirectorv2.redirect_from_cache of <botocore.utils.S3RegionRedirectorv2 object at 0x00000204AC739650>>
2023-02-16 16:18:11,169:DEBUG:botocore.regions:Calling endpoint provider with parameters: {'Bucket': 'arunpdf', 'Region': 'ap-northeast-1', 'UseFIPS': False, 'UseDualStack': False, 'ForcePathStyle': False, 'Accelerate': False, 'UseGlobalEndpoint': False, 'DisableMultiRegionAccessPoints': False, 'UseArnRegion': True}
2023-02-16 16:18:11,169:DEBUG:botocore.regions:Endpoint provider result: https://arunpdf.s3.ap-northeast-1.amazonaws.com
2023-02-16 16:18:11,169:DEBUG:botocore.regions:Selecting from endpoint provider's list of auth schemes: "sigv4". User selected auth scheme is: "None"
2023-02-16 16:18:11,169:DEBUG:botocore.regions:Selected auth type "v4" as "v4" with signing context params: {'region': 'ap-northeast-1', 'signing_name': 's3', 'disableDoubleEncoding': True}
2023-02-16 16:18:11,169:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function validate_ascii_metadata at 0x00000204ABE022A0>
2023-02-16 16:18:11,169:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function sse_md5 at 0x00000204ABE01580>
2023-02-16 16:18:11,169:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function convert_body_to_file_like_object at 0x00000204ABE02CA0>
2023-02-16 16:18:11,169:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function validate_bucket_name at 0x00000204ABE014E0>
2023-02-16 16:18:11,169:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function remove_bucket_from_url_paths_from_model at 0x00000204ABE03600>
2023-02-16 16:18:11,170:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <bound method S3RegionRedirectorv2.annotate_request_context of <botocore.utils.S3RegionRedirectorv2 object at 0x00000204AC739650>>
2023-02-16 16:18:11,170:DEBUG:botocore.hooks:Event before-parameter-build.s3.PutObject: calling handler <function generate_idempotent_uuid at 0x00000204ABE01300>
2023-02-16 16:18:11,170:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function conditionally_calculate_md5 at 0x00000204ABC6C360>
2023-02-16 16:18:11,175:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function add_expect_header at 0x00000204ABE018A0>
2023-02-16 16:18:11,175:DEBUG:botocore.handlers:Adding expect 100 continue header to request.
2023-02-16 16:18:11,175:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function add_recursion_detection_header at 0x00000204ABDEFBA0>
2023-02-16 16:18:11,175:DEBUG:botocore.hooks:Event before-call.s3.PutObject: calling handler <function inject_api_version_header_if_needed at 0x00000204ABE02DE0>
2023-02-16 16:18:11,175:DEBUG:botocore.endpoint:Making request for OperationModel(name=PutObject) with params: {'url_path': '/62d6d281ff20c4b7c05d1e37.pdf', 'query_string': {}, 'method': 'PUT', 'headers': {'User-Agent': 'Boto3/1.26.71 Python/3.11.2 Windows/10 Botocore/1.29.71', 'Content-MD5': 'FzY+9Eqq+VHWNrm4BZvQJA==', 'Expect': '100-continue'}, 'body': <s3transfer.utils.ReadFileChunk object at 0x00000204ACFD24D0>, 'auth_path': '/arunpdf/62d6d281ff20c4b7c05d1e37.pdf', 'url': 'https://arunpdf.s3.ap-northeast-1.amazonaws.com/62d6d281ff20c4b7c05d1e37.pdf', 'context': {'client_region': 'ap-northeast-1', 'client_config': <botocore.config.Config object at 0x00000204AC31C2D0>, 'has_streaming_input': True, 'auth_type': 'v4', 'signing': {'region': 'ap-northeast-1', 'signing_name': 's3', 'disableDoubleEncoding': True}, 's3_redirect': {'redirected': False, 'bucket': 'arunpdf', 'params': {'Bucket': 'arunpdf', 'Key': '62d6d281ff20c4b7c05d1e37.pdf', 'Body': <s3transfer.utils.ReadFileChunk object at 0x00000204ACFD24D0>}}}}
2023-02-16 16:18:11,175:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function signal_not_transferring at 0x00000204ACAA2200>
2023-02-16 16:18:11,175:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <bound method RequestSigner.handler of <botocore.signers.RequestSigner object at 0x00000204ACB160D0>>
2023-02-16 16:18:11,175:DEBUG:botocore.hooks:Event choose-signer.s3.PutObject: calling handler <bound method ClientCreator._default_s3_presign_to_sigv2 of <botocore.client.ClientCreator object at 0x00000204AC30BB50>>
2023-02-16 16:18:11,176:DEBUG:botocore.hooks:Event choose-signer.s3.PutObject: calling handler <function set_operation_specific_signer at 0x00000204ABE011C0>
2023-02-16 16:18:11,176:DEBUG:botocore.hooks:Event before-sign.s3.PutObject: calling handler <function remove_arn_from_signing_path at 0x00000204ABE03740>
2023-02-16 16:18:11,176:DEBUG:botocore.auth:Calculating signature using v4 auth.
2023-02-16 16:18:11,176:DEBUG:botocore.auth:CanonicalRequest:
PUT
/62d6d281ff20c4b7c05d1e37.pdf

content-md5:FzY+9Eqq+VHWNrm4BZvQJA==
host:arunpdf.s3.ap-northeast-1.amazonaws.com
x-amz-content-sha256:UNSIGNED-PAYLOAD
x-amz-date:20230216T104811Z

content-md5;host;x-amz-content-sha256;x-amz-date
UNSIGNED-PAYLOAD
2023-02-16 16:18:11,176:DEBUG:botocore.auth:StringToSign:
AWS4-HMAC-SHA256
20230216T104811Z
20230216/ap-northeast-1/s3/aws4_request
2d2d165737d8b4a6c3fcb30812c11c7060ce43f6ed7ac7beb940c2b10c3eb485
2023-02-16 16:18:11,176:DEBUG:botocore.auth:Signature:
ce3d16bbace5b1d290fe092943333feb860ce50849145e622a3890f527a633f7
2023-02-16 16:18:11,176:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function signal_transferring at 0x00000204ACAA2520>
2023-02-16 16:18:11,176:DEBUG:botocore.hooks:Event request-created.s3.PutObject: calling handler <function add_retry_headers at 0x00000204ABE03560>
2023-02-16 16:18:11,176:DEBUG:botocore.endpoint:Sending http request: <AWSPreparedRequest stream_output=False, method=PUT, url=https://arunpdf.s3.ap-northeast-1.amazonaws.com/62d6d281ff20c4b7c05d1e37.pdf, headers={'User-Agent': b'Boto3/1.26.71 Python/3.11.2 Windows/10 Botocore/1.29.71', 'Content-MD5': b'FzY+9Eqq+VHWNrm4BZvQJA==', 'Expect': b'100-continue', 'X-Amz-Date': b'20230216T104811Z', 'X-Amz-Content-SHA256': b'UNSIGNED-PAYLOAD', 'Authorization': b'AWS4-HMAC-SHA256 Credential=AKIAZIH2ZW4NDK3AU2M6/20230216/ap-northeast-1/s3/aws4_request, SignedHeaders=content-md5;host;x-amz-content-sha256;x-amz-date, Signature=ce3d16bbace5b1d290fe092943333feb860ce50849145e622a3890f527a633f7', 'amz-sdk-invocation-id': b'b081a267-8153-415b-812c-a09959d9db7f', 'amz-sdk-request': b'attempt=1', 'Content-Length': '1744'}>
2023-02-16 16:18:11,177:DEBUG:botocore.httpsession:Certificate path: C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\botocore\cacert.pem
2023-02-16 16:18:11,177:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): arunpdf.s3.ap-northeast-1.amazonaws.com:443
2023-02-16 16:18:11,731:DEBUG:botocore.awsrequest:Waiting for 100 Continue response.
2023-02-16 16:18:11,919:DEBUG:botocore.awsrequest:100 Continue response seen, now sending request body.
2023-02-16 16:18:12,103:DEBUG:urllib3.connectionpool:https://arunpdf.s3.ap-northeast-1.amazonaws.com:443 "PUT /62d6d281ff20c4b7c05d1e37.pdf HTTP/1.1" 200 0
2023-02-16 16:18:12,103:DEBUG:botocore.parsers:Response headers: {'x-amz-id-2': 'YbXqi2f+GAJe/ZRhejA9Y07wwdkS0Bi0pYSx07+RNQJ5Z7y0xJ7oH4ugmRYDv5MlNDHCgPX12gE=', 'x-amz-request-id': 'ANJ5WXPCTM1NB39Z', 'Date': 'Thu, 16 Feb 2023 10:48:14 GMT', 'x-amz-server-side-encryption': 'AES256', 'ETag': '"17363ef44aaaf951d636b9b8059bd024"', 'Server': 'AmazonS3', 'Content-Length': '0'}
2023-02-16 16:18:12,103:DEBUG:botocore.parsers:Response body:
b''
2023-02-16 16:18:12,104:DEBUG:botocore.hooks:Event needs-retry.s3.PutObject: calling handler <botocore.retryhandler.RetryHandler object at 0x00000204ACF8BDD0>
2023-02-16 16:18:12,104:DEBUG:botocore.retryhandler:No retry needed.
2023-02-16 16:18:12,104:DEBUG:botocore.hooks:Event needs-retry.s3.PutObject: calling handler <bound method S3RegionRedirectorv2.redirect_from_error of <botocore.utils.S3RegionRedirectorv2 object at 0x00000204AC739650>>
2023-02-16 16:18:12,104:DEBUG:s3transfer.utils:Releasing acquire 0/None
2023-02-16 16:18:12,106:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:18:12] "POST /course_detail HTTP/1.1" 200 -
2023-02-16 16:18:25,224:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:18:25,226:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:18:25] "POST /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:18:26,623:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:18:26,626:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:18:26] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:18:29,481:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:18:29,482:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:18:29] "GET /home_page HTTP/1.1" 200 -
2023-02-16 16:18:31,337:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:18:31,339:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:18:31] "POST /home_page HTTP/1.1" 200 -
2023-02-16 16:18:31,423:INFO:root:MongoDB Connection...Done
2023-02-16 16:18:31,423:INFO:root:MongoDB Database...Done
2023-02-16 16:18:31,423:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 16:18:31,423:INFO:root:MongoDB Collection courses ...Created
2023-02-16 16:18:32,660:ERROR:root:batch op errors occurred, full error: {'writeErrors': [{'index': 0, 'code': 11000, 'keyPattern': {'_id': 1}, 'keyValue': {'_id': ObjectId('63ee097657a629d0acfb2e0e')}, 'errmsg': "E11000 duplicate key error collection: ineuron.cat_subcat index: _id_ dup key: { _id: ObjectId('63ee097657a629d0acfb2e0e') }", 'op': {'cat_id': '62cd958353f5fe61566b9ae4', 'cat_name': 'DATA SCIENCE', 'sub_cat_id': '62cd959d53f5fe14766b9ae5', 'sub_cat_name': 'FULL STACK DATA SCIENCE', '_id': ObjectId('63ee097657a629d0acfb2e0e')}}], 'writeConcernErrors': [], 'nInserted': 0, 'nUpserted': 0, 'nMatched': 0, 'nModified': 0, 'nRemoved': 0, 'upserted': []}
2023-02-16 16:18:32,660:INFO:root:MongoDB Data Insert...Failed
2023-02-16 16:18:34,662:ERROR:root:batch op errors occurred, full error: {'writeErrors': [{'index': 0, 'code': 11000, 'keyPattern': {'_id': 1}, 'keyValue': {'_id': ObjectId('63ee097757a629d0acfb2ec4')}, 'errmsg': "E11000 duplicate key error collection: ineuron.courses index: _id_ dup key: { _id: ObjectId('63ee097757a629d0acfb2ec4') }", 'op': {'course_id': '63bbd0863d8c5c05632becbc', 'course_name': 'Complete DSA in Python', 'course_description': 'A comprehensive chase to excel any intreview for the Data Structures and Algorithms. This course has been specifically designed to provide resources that would assist you in cracking problem-solving interviews. The presented problems in the course would suffice to look on to positive outcomes in the interviews.', 'course_mode': 'self-paced', 'course_cat_id': '62cda11e53f5fe7a386b9b87', 'course_timings': {'timings': 'Flexible'}, 'course_curriculum': ['Introduction to Algorithms', 'Analysis in Algorithms', 'Array Data Structure', 'Heap Data Structure', 'Recursion', 'Divide and Conquer', 'Linked List Data Structure', 'Stack and Queue', 'Hashing Data Structure', 'Tree Data Structure', 'Binary Search Tree', 'Graph Traversal Algorithms', 'Application of greedy algorithm', 'Dynamic Programming', 'Research Area- P, NP, NP-Hard and NP-Complete Problems'], 'course_requirements': ['System with minimum i3 processor or better', 'At least 4 GB of RAM', 'Working internet connection', 'Dedication to learn'], 'course_features': ['Course Materials', 'Self Paced Learning', 'Lifetime Dashboard Access', 'Completion Certificate'], 'instructor_id': ['618964776a74763ac407df8b'], 'instructor_name': ['Priya Bhatia'], 'instructors_social': [{'linkedin': 'https://www.linkedin.com/in/bhatia-priya/', 'instagram': 'https://www.instagram.com/priyabhatia6971/', 'facebook': 'https://www.facebook.com/priya.bhatia.37604', 'github': 'https://github.com/priya6971'}], 'instructor_description': ['Expertise in data structure competitive programing and solving an analytical problems and implementing data structure algorithm in multiple programing language. I have done my M.Tech in Artificial Intelligence at IIT Hyderabad and have an experience of implementation in multiple projects.'], '_id': ObjectId('63ee097757a629d0acfb2ec4')}}], 'writeConcernErrors': [], 'nInserted': 0, 'nUpserted': 0, 'nMatched': 0, 'nModified': 0, 'nRemoved': 0, 'upserted': []}
2023-02-16 16:18:34,662:INFO:root:MongoDB Data Insert...Failed
2023-02-16 16:18:35,345:INFO:root:Connected to MySQL Database...
2023-02-16 16:18:35,678:INFO:root:Created Database...iNeuronDb
2023-02-16 16:18:35,834:INFO:root:Using Database...iNeuronDb
2023-02-16 16:18:36,006:INFO:root:Created Table...courses
2023-02-16 16:18:36,006:INFO:root:Total data to Insert: 450
2023-02-16 16:19:52,639:INFO:root:Total data Inserted in Table: 1
2023-02-16 16:19:52,640:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:19:52,640:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:19:52] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:19:52,654:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:19:52,657:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:19:52] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:19:58,266:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:19:58,267:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:19:58] "GET /home_page HTTP/1.1" 200 -
2023-02-16 16:24:46,353:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 16:24:46,904:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:46,904:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 16:24:48,835:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:48,835:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 16:24:48,835:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:48,835:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 16:24:48,835:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:48,835:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 16:24:48,884:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:48,884:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 16:24:48,884:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:48,884:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 16:24:48,884:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:48,884:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 16:24:48,884:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:48,884:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 16:24:48,916:DEBUG:NeuronScrapper:Done
2023-02-16 16:24:48,932:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 16:24:48,932:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 16:24:58,465:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:24:58,465:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:24:58] "GET / HTTP/1.1" 200 -
2023-02-16 16:25:02,797:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:25:02,799:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:25:02] "POST /home_page HTTP/1.1" 200 -
2023-02-16 16:25:02,897:INFO:root:MongoDB Connection...Done
2023-02-16 16:25:02,897:INFO:root:MongoDB Database...Done
2023-02-16 16:25:02,897:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 16:25:02,897:INFO:root:MongoDB Collection courses ...Created
2023-02-16 16:25:04,254:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:25:06,219:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:25:06,932:INFO:root:Connected to MySQL Database...
2023-02-16 16:25:07,245:INFO:root:Created Database...iNeuronDb
2023-02-16 16:25:07,401:INFO:root:Using Database...iNeuronDb
2023-02-16 16:25:07,557:INFO:root:Created Table...courses
2023-02-16 16:25:07,557:INFO:root:Total data to Insert: 450
2023-02-16 16:26:20,868:INFO:root:Total data Inserted in Table: 1
2023-02-16 16:26:20,868:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:26:20,869:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:26:20] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:26:20,882:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:26:20,888:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:26:20] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:27:07,626:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:27:07,627:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:27:07] "GET /home_page HTTP/1.1" 200 -
2023-02-16 16:29:03,677:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 16:29:03,881:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:03,881:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 16:29:07,018:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:07,018:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 16:29:07,018:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:07,018:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 16:29:07,018:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:07,018:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 16:29:07,049:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:07,049:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 16:29:07,049:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:07,049:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 16:29:07,049:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:07,049:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 16:29:07,049:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:07,049:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 16:29:07,081:DEBUG:NeuronScrapper:Done
2023-02-16 16:29:07,096:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 16:29:07,096:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 16:29:22,456:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:29:22,462:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:29:22] "GET / HTTP/1.1" 200 -
2023-02-16 16:29:26,991:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:29:26,994:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:29:26] "POST /home_page HTTP/1.1" 200 -
2023-02-16 16:29:27,069:INFO:root:MongoDB Connection...Done
2023-02-16 16:29:27,069:INFO:root:MongoDB Database...Done
2023-02-16 16:29:27,069:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 16:29:27,069:INFO:root:MongoDB Collection courses ...Created
2023-02-16 16:29:28,376:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:29:30,353:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:29:31,062:INFO:root:Connected to MySQL Database...
2023-02-16 16:29:31,406:INFO:root:Created Database...iNeuronDb
2023-02-16 16:29:31,578:INFO:root:Using Database...iNeuronDb
2023-02-16 16:29:31,766:ERROR:root:(1170, "BLOB/TEXT column 'course_id' used in key specification without a key length")
2023-02-16 16:29:31,766:INFO:root:Error occurred while creating Table: courses
2023-02-16 16:29:31,766:INFO:root:Total data to Insert: 450
2023-02-16 16:29:32,102:ERROR:root:(1146, "Table 'iNeuronDb.courses' doesn't exist")
2023-02-16 16:29:32,102:INFO:root:Error occurred while Inserting Data in Table
2023-02-16 16:29:32,102:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:29:32,102:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:29:32] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:29:32,125:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:29:32,130:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:29:32] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:31:26,187:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 16:31:26,390:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:26,390:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 16:31:28,221:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:28,221:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 16:31:28,237:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:28,237:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 16:31:28,237:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:28,237:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 16:31:28,253:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:28,253:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 16:31:28,253:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:28,253:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 16:31:28,253:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:28,253:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 16:31:28,253:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:28,253:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 16:31:28,299:DEBUG:NeuronScrapper:Done
2023-02-16 16:31:28,299:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 16:31:28,299:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 16:31:46,225:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:31:46,231:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:31:46] "GET /home_page HTTP/1.1" 200 -
2023-02-16 16:31:48,952:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:31:48,953:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:31:48] "GET / HTTP/1.1" 200 -
2023-02-16 16:31:59,201:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:31:59,204:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:31:59] "POST /home_page HTTP/1.1" 200 -
2023-02-16 16:31:59,355:INFO:root:MongoDB Connection...Done
2023-02-16 16:31:59,355:INFO:root:MongoDB Database...Done
2023-02-16 16:31:59,356:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 16:31:59,356:INFO:root:MongoDB Collection courses ...Created
2023-02-16 16:32:00,599:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:32:03,780:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:32:04,497:INFO:root:Connected to MySQL Database...
2023-02-16 16:32:04,854:INFO:root:Created Database...iNeuronDb
2023-02-16 16:32:05,056:INFO:root:Using Database...iNeuronDb
2023-02-16 16:32:05,261:ERROR:root:(1170, "BLOB/TEXT column 'course_id' used in key specification without a key length")
2023-02-16 16:32:05,261:INFO:root:Error occurred while creating Table: courses
2023-02-16 16:32:05,261:INFO:root:Total data to Insert: 450
2023-02-16 16:32:05,596:ERROR:root:(1146, "Table 'iNeuronDb.courses' doesn't exist")
2023-02-16 16:32:05,596:INFO:root:Error occurred while Inserting Data in Table
2023-02-16 16:32:05,596:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:32:05,597:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:32:05] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:32:05,608:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:32:05,614:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:32:05] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:36:45,931:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 16:36:46,175:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:46,175:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 16:36:48,241:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:48,241:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 16:36:48,252:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:48,252:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 16:36:48,253:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:48,253:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 16:36:48,276:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:48,276:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 16:36:48,277:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:48,277:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 16:36:48,279:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:48,279:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 16:36:48,281:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:48,281:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 16:36:48,317:DEBUG:NeuronScrapper:Done
2023-02-16 16:36:48,324:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 16:36:48,324:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 16:36:50,370:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:36:50,376:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:36:50] "GET / HTTP/1.1" 200 -
2023-02-16 16:36:52,176:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:36:52,179:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:36:52] "POST /home_page HTTP/1.1" 200 -
2023-02-16 16:36:52,281:INFO:root:MongoDB Connection...Done
2023-02-16 16:36:52,281:INFO:root:MongoDB Database...Done
2023-02-16 16:36:52,281:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 16:36:52,281:INFO:root:MongoDB Collection courses ...Created
2023-02-16 16:36:53,539:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:36:55,588:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:36:56,220:INFO:root:Connected to MySQL Database...
2023-02-16 16:36:56,526:INFO:root:Created Database...iNeuronDb
2023-02-16 16:36:56,676:INFO:root:Using Database...iNeuronDb
2023-02-16 16:36:56,864:INFO:root:Created Table...courses
2023-02-16 16:36:56,864:INFO:root:Total data to Insert: 450
2023-02-16 16:38:06,821:INFO:root:Total data Inserted in Table: 1
2023-02-16 16:38:06,821:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:38:06,822:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:38:06] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:38:06,833:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:38:06,840:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:38:06] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:39:09,410:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:39:09,412:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:39:09] "GET / HTTP/1.1" 200 -
2023-02-16 16:39:11,472:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:39:11,473:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:39:11] "POST /home_page HTTP/1.1" 200 -
2023-02-16 16:39:11,550:INFO:root:MongoDB Connection...Done
2023-02-16 16:39:11,550:INFO:root:MongoDB Database...Done
2023-02-16 16:39:11,550:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 16:39:11,550:INFO:root:MongoDB Collection courses ...Created
2023-02-16 16:39:12,888:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:39:15,916:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:39:16,689:INFO:root:Connected to MySQL Database...
2023-02-16 16:39:17,033:INFO:root:Created Database...iNeuronDb
2023-02-16 16:39:17,204:INFO:root:Using Database...iNeuronDb
2023-02-16 16:39:17,397:INFO:root:Created Table...courses
2023-02-16 16:39:17,397:INFO:root:Total data to Insert: 450
2023-02-16 16:39:17,770:ERROR:root:(1062, "Duplicate entry '63bbd0863d8c5c05632becbc' for key 'courses.PRIMARY'")
2023-02-16 16:39:17,770:INFO:root:Error occurred while Inserting Data in Table
2023-02-16 16:39:17,770:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:39:17,771:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:39:17] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:39:17,783:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:39:17,785:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:39:17] "GET /cat_subcat_list HTTP/1.1" 200 -
2023-02-16 16:42:34,355:DEBUG:NeuronScrapper:Getting Webpage...
2023-02-16 16:42:34,621:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:34,621:DEBUG:NeuronScrapper:Reading Webpage...
2023-02-16 16:42:36,812:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:36,812:DEBUG:NeuronScrapper:Converting Webpage to HTML...
2023-02-16 16:42:36,828:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:36,828:DEBUG:NeuronScrapper:Extracting Raw Data...
2023-02-16 16:42:36,828:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:36,832:DEBUG:NeuronScrapper:Converting to JSON...
2023-02-16 16:42:36,843:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:36,843:DEBUG:NeuronScrapper:Getting Category and Subcategory Data...
2023-02-16 16:42:36,843:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:36,843:DEBUG:NeuronScrapper:Saving cat_sub_cat_data.json
2023-02-16 16:42:36,843:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:36,843:DEBUG:NeuronScrapper:Getting Courses Data...
2023-02-16 16:42:36,859:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:36,859:DEBUG:NeuronScrapper:Saving courses_data.json
2023-02-16 16:42:36,896:DEBUG:NeuronScrapper:Done
2023-02-16 16:42:36,903:INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
2023-02-16 16:42:36,903:INFO:werkzeug:[33mPress CTRL+C to quit[0m
2023-02-16 16:42:42,337:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:42:42,343:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:42:42] "GET / HTTP/1.1" 200 -
2023-02-16 16:42:53,547:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:42:53,550:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:42:53] "POST /home_page HTTP/1.1" 200 -
2023-02-16 16:42:53,642:INFO:root:MongoDB Connection...Done
2023-02-16 16:42:53,642:INFO:root:MongoDB Database...Done
2023-02-16 16:42:53,642:INFO:root:MongoDB Collection cat_subcat ...Created
2023-02-16 16:42:53,642:INFO:root:MongoDB Collection courses ...Created
2023-02-16 16:42:54,888:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:42:56,998:INFO:root:MongoDB Data Insert ...Done
2023-02-16 16:42:57,765:INFO:root:Connected to MySQL Database...
2023-02-16 16:42:58,093:INFO:root:Created Database...iNeuronDb
2023-02-16 16:42:58,249:INFO:root:Using Database...iNeuronDb
2023-02-16 16:42:58,406:INFO:root:Created Table...courses
2023-02-16 16:42:58,406:INFO:root:Total data to Insert: 450
2023-02-16 16:44:12,902:INFO:root:Total data Inserted in Table: 1
2023-02-16 16:44:12,902:ERROR:app:Exception on /load_data [GET]
Traceback (most recent call last):
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2525, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1823, in full_dispatch_request
    return self.finalize_request(rv)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 1842, in finalize_request
    response = self.make_response(rv)
               ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\arunl\AppData\Local\Programs\Python\Python311\Lib\site-packages\flask\app.py", line 2134, in make_response
    raise TypeError(
TypeError: The view function for 'load_data' did not return a valid response. The function either returned None or ended without a return statement.
2023-02-16 16:44:12,902:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:44:12] "[35m[1mGET /load_data HTTP/1.1[0m" 500 -
2023-02-16 16:44:12,919:DEBUG:asyncio:Using proactor: IocpProactor
2023-02-16 16:44:12,925:INFO:werkzeug:127.0.0.1 - - [16/Feb/2023 16:44:12] "GET /cat_subcat_list HTTP/1.1" 200 -
